### ISSUE TITLE
Intel G33/G31 DirectX 9.0c (Broken support)

### ISSUE THREAD

**Issue by MarkWolves** [hrydgard/ppsspp#18469]
### Game or games this happens in

The emulator does not open to perform this test step.

### What area of the game

The emulator does not open to perform this test step.

### What happens

PPSSPP version 1.14.x above does not work on Intel G33/G31 (DirectX 9.0c) graphics cards.

Look at the gif image:
![GIF](https://github.com/hrydgard/ppsspp/assets/141780990/bc61c701-a9b8-42b4-aef3-00d6f0662040)

### What should happen

New versions above 1.14.x are expected to support the Intel G33/G31 version as well as 1.13.x versions.

Is it a bug or support drop?

### GE frame capture

Not necessary

### Platform

Windows

### Mobile phone model or graphics card

Intel(R) G33/G31 Express Chipset Family

### PPSSPP version affected

v1.14.x up to v1.16.6

### Last working version

v1.13.2

### Graphics backend (3D API)

Direct3D 9

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the glitch (especially speed hacks or enhancements/replacements.)
- [X] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
Likely an accidental regressions. Hard to say exactly what it could be, I don't have any machines with any of these old-generation Intel GPUs to test on...

Someone would need to run the latest PPSSPP in the MSVC debugger on a machine with this problem and see where it crashes.

**Comment by MarkWolves** [comment]
I've already tried updating with "DirectX Web Setup".

**Comment by MarkWolves** [comment]
No. Its opengl is 1.4 (bear lake b) on Windows.On Linux, since the opengl driver is 2.1, it works.Em 7 de mar de 2025 10:05, mateli ***@***.***> escreveu:
  Does the OpenGL backend work?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>



mateli left a comment (hrydgard/ppsspp#18469)
Does the OpenGL backend work?

**Comment by hrydgard** [comment]
Is it possible for you to use D3D11?

**Comment by MarkWolves** [comment]
No.Em 11 de mar de 2025 10:29, Henrik Rydgård ***@***.***> escreveu:
  Is it possible for you to use D3D11?—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>



hrydgard left a comment (hrydgard/ppsspp#18469)
Is it possible for you to use D3D11?

**Comment by mateli** [comment]
> No. Its opengl is 1.4 (bear lake b) on Windows.On Linux, since the opengl driver is 2.1, it works.

Then perhaps stick to the OS that has the best and most up to date drivers for your hardware.

**Comment by hrydgard** [comment]
DX9 is gone, closing.

**Comment by enriquepene1221-sys** [comment]
Se debe al shader model.

La GPU Intel G33/G31 solo soporta shader model 2.0, en cambio, PPSSPP desde la 1.14.x usa shader model 3.0 (aunque ya no es util saber esto)
